open-company/open-company-email
# Transactional email updates

Card: https://trello.com/c/AgMB43JG

To test:
- check the templates of the following emails
- invite a user via email
- invite a user to a private board
- Reset password
- Email verification
- Reminder a user about a post
- User X commented on your post
- User X also commented
- User X mentioned you in a post
- User X mentioned you in a comment
- Reminder created
- Reminder due
- Request follow up
- Digest (this wasn't updated, just check regressions)
